README.md

```diff
@@ -1 +1,32 @@
-# Fall2020_CSC403_Project
\ No newline at end of file
+# Team CSC403-002 22F Scrum Project
+
+## Documentation 
+
+### Ingame Pause Window 
+
+Created a new Windows Forms class that acts as the pause menu for the game called `FrmPause.cs`. A `FrmPause` instance is initiated and shown when the player presses the escape button. Unlike a `FrmBattle` instance, a player can not continue to play the game while a `FrmPause` window is shown. The reason is because inside `FrmLevel.cs`, frmBattle, an instance of `FrmBattle.cs`, uses `frmBattle.show()` to show the window while frmPause, an instance of `FrmPause.cs`, uses `frmPause.ShowDialog()` which does not allow the player to access any other windows while frmPause is shown. 
+
+A boolean class variable inside of `FrmLevel.cs` called isPaused was created solely for the purpose of stopping the ingame timer. isPaused's value is initially set to false. When the escape key is hit, the isPaused variable is set to true, and when the frmPause window is closed, isPaused is set back to false. More on pausing the ingame timer is discussed below.
+
+### `FrmPause.cs` Functionality 
```

Comment:
This line is on the same level of indentation as our names - might be confusing. 
```suggestion
#### `FrmPause.cs` Functionality 
```